test/unit/translators/twitter_translators/user_translator_test.rb

```diff
@@ -20,7 +20,7 @@ def build_twitter_user
     end
   end
 
-  test "no new user created after translating an existing user" do
+  test 'no new user created after translating an existing user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.